src/Configuration/src/ConfigServer/Steeltoe.Configuration.ConfigServer.csproj

```diff
@@ -18,8 +18,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\src\Common.Hosting\Steeltoe.Common.Hosting.csproj" />
     <ProjectReference Include="..\..\..\Common\src\Common.Http\Steeltoe.Common.Http.csproj" />
     <ProjectReference Include="..\CloudFoundry\Steeltoe.Configuration.CloudFoundry.csproj" />
+    <ProjectReference Include="..\Kubernetes.ServiceBinding\Steeltoe.Configuration.Kubernetes.ServiceBinding.csproj" />
     <ProjectReference Include="..\Placeholder\Steeltoe.Configuration.Placeholder.csproj" />
     <ProjectReference Include="..\..\..\Discovery\src\Client\Steeltoe.Discovery.Client.csproj" PrivateAssets="All" />
```

Comment:
Why is this a private reference? Isn't that just satisfying the compiler but causing a runtime error when the package isn't loaded? It's needed by `ConfigServerDiscoveryService`, but that method isn't decorated with `[MethodImpl(MethodImplOptions.NoInlining)]` like it was in the Bootstrap project.